spring-vault-core/pom.xml

```diff
@@ -75,6 +75,11 @@
 			<artifactId>jackson-databind</artifactId>
 		</dependency>
 
+		<dependency>
+			<groupId>com.fasterxml.jackson.datatype</groupId>
```

Comment:
We should rather not pull in the JSR-310 support here as this dependency change has a few downstream consequences (such as customizing the `ObjectMapper`).